Update pull_request_template.md to reference data quality

howaskew opened the pull request:
## Publishing Checklist

I confirm the following:

- [ ] My open data feeds conform to one of the permissable combinations specified in https://developer.openactive.io/publishing-data/data-feeds/types-of-feed
- [ ] My open data feeds all pass the OpenActive Validator's model validation (https://validator.openactive.io/)
- [ ] My open data feeds all pass the OpenActive Validator's RPDE feed validation (https://validator.openactive.io/rpde)
- [ ] My open data feeds include activity list references as specified in https://developer.openactive.io/publishing-data/activity-list-references
- [ ] My dataset site(s) each reference a GitHub Issues Board

## Maintenance Checklist

Our organisation is committed to:

- [ ] Resolving issues that are raised on the GitHub Issues Board(s)
- [ ] Ensuring activity providers are aware of the OpenActive opportunity on an ongoing basis

howaskew committed:
Update pull_request_template.md to reference data quality

nickevansuk commented:
I'd suggest putting a summary of the key points of that data quality doc on https://developer.openactive.io, and then linking to a URL there - rather than a link to a Google doc directly. We've tried to avoid links to Google Docs throughout the infra and favoured stable links to URLs (noting that GitBooks automatically redirects if the URL of a page changes over time).

Additionally, the Google doc seems to be aimed at a different audience to those here; the audience here would likely benefit from a succinct summary of the "Initial Focus" and "Defining ‘good’ data quality" sections which answers the question "What makes my data high quality?", without the dashboard etc around it.

The documentation is intended to be immediately actionable for readers who are short on time and are seeking the key information they need, so shorter is better.

Finally, the Google Doc - if it is referenced in the developer documentation as suggested above - should probably live here: https://drive.google.com/drive/folders/11xwaofr1G5q_P9u54BgJch9RHLZr7SA4?usp=drive_link

howaskew committed:
Update pull_request_template.md

Linking to developer docs

howaskew commented:
Thanks @nickevansuk So, I've:

- Added a brief data quality section on developer docs, that contains the link to a google doc
- Moved the doc to the correct place on the OA google drive
- Updated this change to point to the developer docs, not the google docs/drive

I will update the doc with latest on DQ framework, but the above are sufficient for this immediate change I think.
